dev-python/nextinspace/nextinspace-3.0.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=poetry
+PYTHON_COMPAT=( python3_{10..13} )
+inherit distutils-r1
+
+DESCRIPTION="A command-line tool for seeing the latest in space"
+HOMEPAGE="https://github.com/gideonshaked/nextinspace"
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/gideonshaked/nextinspace.git"
+else
+	SRC_URI="https://github.com/gideonshaked/nextinspace/archive/refs/tags/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3+"
+SLOT="0"
+
+RDEPEND="
+	>=dev-python/colorama-0.4.3[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.24[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	test? (
+		${RDEPEND}
+		dev-python/pytest-lazy-fixtures[${PYTHON_USEDEP}]
+		>=dev-python/requests-mock-1.8[${PYTHON_USEDEP}]
+	)
+"
+
+distutils_enable_tests pytest
+
+src_prepare() {
+	default
+
+	# Don't install license files
+	sed -e '/^include = \["LICENSE"\]$/d' -i pyproject.toml || die
```

Comment:
Why isn't there a pull request upstream to fix this?

Reply:
I wasn't aware the issue behind that fix is a general type of error. I submitted a fix to upstream and learned a bit more about python packages by the way :smiley:

Reply:
Wait, you've never seen my "Poetry is a true horror" series of toots? Perhaps it's time to put them in a blog post.